sema/commons/rdf_ogm/object_graph_mapper.py

```
@@ -0,0 +1,148 @@
+
+    def create_rnode(self, *args, **kwargs):
+        return self.create_node(*args, **kwargs, kind="relative")
+
+    def create_bnode(self, *args, **kwargs):
+        return self.create_node(*args, **kwargs)
+    
+    def create_node(
+            self,
+            identifier: Union[URIRef, str] = None,
+            a: Union[URIRef, str, list[Union[URIRef, str]]] = None,
+            label: Union[str, list[str]] = None,
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)